ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -51,7 +51,10 @@ export default class ConfirmPageContainerContent extends Component {
     hideTitle: PropTypes.bool,
     supportsEIP1559V2: PropTypes.bool,
     hasTopBorder: PropTypes.bool,
-    currentTransaction: PropTypes.string,
+    currentTransaction: PropTypes.shape({
```

Comment:
This was causing a warning to get generated during e2e tests, I thought I would just fix it so that they're easier to read in the future.

Reply:
I wonder if some might prefer these changes to be in separate PR/commit, I'm not too opinionated on this matter but just calling it out.